SPEC.rdoc

```diff
@@ -132,6 +132,10 @@ There are the following restrictions:
   set. <tt>PATH_INFO</tt> should be <tt>/</tt> if
   <tt>SCRIPT_NAME</tt> is empty.
   <tt>SCRIPT_NAME</tt> never should be <tt>/</tt>, but instead be empty.
+<tt>rack.response_finished</tt>:: An array of callables run after the HTTP response has been finished,
```

Comment:
Maybe `s/finished, either normally or with/fully sent to the client, or there was/`.  I'm not sure whether `finished` is descriptive enough.

Reply:
I'm fine with that, thanks.

Reply:
I've reworked this to clarify in a more abstract sense.